upstash/redis/database/resource.go

```diff
@@ -62,6 +63,7 @@ func ResourceDatabase() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 				Description: "When enabled, data is encrypted in transit. (If changed to false from true, results in deletion and recreation of the resource)",
+				Deprecated:  "TLS option is deprecated.",
```

Comment:
it is default `false`, shouldn't we make it true and only allow `true`..
even user makes it `false` backend will create a database with `true`

Reply:
Did not want to do that, since the default was set as false
Changing the default would meddle with their already generated infra. Instead, made necessary checks such that nothing happens automatically/unwillingly while enforcing the new conventions